src/redux/store.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware, compose } from "redux";
+import thunkMiddleware from "redux-thunk";
+import reducer from "./reducer";
+
+//conectamos nuestra app con la extension
+const composeEnhancer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
If we use redux toolkit? it does not need all this configuration.

Reply:
Good suggestion! Thank you!